deno/payloads/v10/user.ts

```diff
@@ -84,11 +84,11 @@ export interface APIUser {
 	 */
 	public_flags?: UserFlags;
 	/**
-	 * The user's avatar decoration hash
+	 * The data for the user's avatar decoration
 	 *
-	 * See https://discord.com/developers/docs/reference#image-formatting
+	 * See https://discord.com/developers/docs/resources/user#avatar-decoration-data-object
 	 */
-	avatar_decoration?: string | null;
+	avatar_decoration?: AvatarDecorationData | null;
```

Comment:
This is, I assume, now wrong (and a breaking change). `avatar_decoration` should stay as it is but become deprecated. The new property is `avatar_decoration_data`.